content/messaging/agent-chat-api/v3.3/index.mdx

```diff
@@ -118,6 +118,42 @@ The **File** event informs about an uploaded file.
 </Code>
 </Section>
 
+## Form
+
+<Section>
+<Text>
+
+The **Form** event contains data of form that can be filled by customer (sent during agent inactivity for example).
+
+### Request
+
+| Field        | Required | Data type | Notes                                                                    |
+| ------------ | -------- | --------- | ------------------------------------------------------------------------ |
+| `custom_id`  | no       | `string`  |                                                                          |
+| `type`       | yes      | `string`  | `form`                                                                   |
+| `properties` | no       | `object`  | [Properties](#properties)                                                |
+| `form_id`    | yes      | `string`  |                                                                          |
+
+### Response
+
+| Field        | Returned   | Notes                                                               |
+| ------------ | ---------- | ------------------------------------------------------------------- |
+| `id`         | always     |                                                                     |
+| `custom_id`  | optionally |                                                                     |
+| `created_at` | always     | Date & time format with a resolution of microseconds, `UTC string`. |
+| `type`       | always     |                                                                     |
+| `author_id`  | optionally | If form is generated by `system` then this field is omitted.        |
+| `properties` | optionally |                                                                     |
+| `form_id`    | always     |                                                                     |
+| `fields`     | always     | An array of [form fields](#form-fields)                             |
```

Comment:
Watch out, these are not the same fields as filled form fields

Reply:
What are those fields, then? 🤔

Reply:
+ I forgot to delete the description of this field.